articles/quickstart/native/flutter/interactive.md

```diff
@@ -0,0 +1,127 @@
+- `com_auth0_scheme`: The scheme to use. Can be a custom scheme, or `https` if you want to use [Android App Links](https://auth0.com/docs/applications/enable-android-app-links). You can read more about setting this value in the [Auth0.Android SDK README](https://github.com/auth0/Auth0.Android#a-note-about-app-deep-linking).
+
+Run **Sync Project with Gradle Files** inside Android Studio to apply your changes.
+
+## Add login to your app {{{ data-action="code" data-code="main.dart#4:5" }}}
+
+[Universal Login](https://auth0.com/docs/authenticate/login/auth0-universal-login) is the easiest way to set up authentication in your application. We recommend using it for the best experience, best security, and the fullest array of features.
+
+Integrate with Auth0 Universal Login in your Flutter app by using the `Auth0` class. Redirect your users to the Auth0 Universal Login page using `webAuthentication.login()`. This is a `Future` and must be awaited for you to retrieve the user's tokens.
+
+**Android**: if you are using a custom scheme, you must pass this scheme to the login method so that the SDK can route to the login page and back again correctly:
+
+```dart
+await auth0.webAuthentication.login(scheme: 'YOUR CUSTOM SCHEME');
+```
+
+Once a user has logged in, they will be redirected back to your application. You will then be able to access the ID and access tokens for this user.
+
+::::checkpoint
+:::checkpoint-default
+Add a button to your app that calls `webAuthentication.login()` and logs the user into your app. Verify that you are redirected to Auth0 for authentication, and then back to your application.
+
+Verify that you can get access to the tokens on the result of calling `login`.
+:::
+
+:::checkpoint-failure
+If your application did not launch successfully:
+* Ensure you set the Allowed Callback URLs are correct
+* Verify you saved your changes after entering your URLs
+* Make sure the domain and client ID values are imported correctly
+* If using Android, ensure that the manifest placeholders have been set up correctly, otherwise the redirect back to your app may not work
+
+Still having issues? Check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com) to get more help.
+:::
+::::
+
+## Show user profile information {{{ data-action="code" data-code="main.dart#8:9" }}}
+
+The user profile is automatically retrieved for you when you call `webAuthentication.login()`. The object that is returned from the login step contains a `userProfile` property that has all the user profile properties, which have been populated by decoding the ID token.
+
+::::checkpoint
+:::checkpoint-default
+Log in and inspect the `userProfile` property on the result. Verify that you can access the current user's profile information.
```

Comment:
Same as above.

Reply:
Should we mention a specific profile property for the developer to try out, like `email`?